### PR TITLE
What happens when CREATE[2] returns an excessively long buffer?

### PR DESCRIPTION
**Pull request by qbzzt**

### CODE REVIEW

File: src/GeneralStateTestsFiller/stCreateTest/createLargeResultFiller.yml [hunk 1]
```diff
@@ -0,0 +1,428 @@
...
+      nonce: 1
+      storage: {}
+
+
+  transaction:
+    data:
+    # "Normal" contract (256 bytes)
+    - :label CREATE-GOOD    :abi f(uint,uint,uint) 0xF0 0xF3 0x100
+    - :label CREATE2-GOOD   :abi f(uint,uint,uint) 0xF5 0xF3 0x100
+    - :label CREATE-FAIL    :abi f(uint,uint,uint) 0xF0 0xFD 0x100
```
Comment (winsvega) [inline src/GeneralStateTestsFiller/stCreateTest/createLargeResultFiller.yml]: CREATE-REVERT

File: src/GeneralStateTestsFiller/stCreateTest/createLargeResultFiller.yml [hunk 2]
```diff
@@ -0,0 +1,428 @@
...
+         case 0xF5 { newAddr := create2(0, 0, 0x120, 0x5A17) }
+
+         // Write the results
+         sstore(0, newAddr)
+         sstore(1, gas())
+         sstore(2, extcodehash(newAddr))
+         
+       }        
+      nonce: 1
+      storage: {}
```
Comment (winsvega) [inline src/GeneralStateTestsFiller/stCreateTest/createLargeResultFiller.yml]: Put some values to storage at 0,1,2 because you check it for 0

**Comment by jochem-brouwer** [comment]
This indeed triggers the issue solved in https://github.com/ethereumjs/ethereumjs-monorepo/pull/1700, thanks :smile:

**Comment by qbzzt** [comment]
> This indeed triggers the issue solved in [ethereumjs/ethereumjs-monorepo#1700](https://github.com/ethereumjs/ethereumjs-monorepo/pull/1700), thanks 😄

Excellent, that's the effect I was going for.

**Pull request merged**